app/routes/authentication.py

```diff
@@ -95,6 +95,35 @@ def register_user():
     return "", 204
 
 
+@auth_blueprint.route("/change-password", methods=["POST"])
```

Comment:
If they are changing their password they should have a jwt token. So maybe make the input just the two passwords and get email from jwt token.

```
@auth_blueprint.route("/change-password", methods=["POST"])
@jwt_required
def change_password():
    current_user = User.query.filter_by(email=get_jwt_identity()).first()
    old_password = request.json.get("old_password", None)
    new_password = request.json.get("new_password", None)
    ....
```

Reply:
I don't think that's necessarily true.  What if it was a new machine they were trying to log in on and they couldn't remember the password.